packages/tpetra/core/compat/Tpetra_KokkosCompat_ClassicNodeAPI_Wrapper.hpp

```diff
@@ -43,6 +43,18 @@ class KokkosDeviceWrapperNode {
   /// release.  This Node type is safe to use.
   static constexpr bool classic = false;
 
+  //! Whether the ExecutionSpace is Kokkos::Serial.
+#ifdef KOKKOS_ENABLE_SERIAL
+  static constexpr bool is_serial = std::is_same_v<ExecutionSpace, Kokkos::Serial>;
+#else
+  static constexpr bool is_serial = false;
+#endif
+
+  //! Whether the ExecutionSpace is CPU-like (its default memory space is HostSpace)
+  static constexpr bool is_cpu = std::is_same_v<typename ExecutionSpace::memory_space, Kokkos::HostSpace>;
```

Comment:
Careful, Kokkos actually also defines `HBWSpace` for things like KNL and SapphireRapids which is should lead to `is_cpu == true`.

Reply:
Good catch, I'll fix this in a follow up PR